### PR TITLE
Rename centos-stream centos-8-stream

### PR DESCRIPTION
**Pull request by opoplawski**
We're going to have centos-9-stream, etc..

### CODE REVIEW

**Comment by praiskup** [comment]
I think this is likely, but do you have an official statement this is going to happen?

**Comment by Conan-Kudo** [comment]
@praiskup @carlwgeorge has confirmed that is what is happening.

**Comment by praiskup** [comment]
Ok, thanks.  @opoplawski can you please change the name to `centos-stream-8-<arch>`?

**Comment by carlwgeorge** [comment]
> I think this is likely, but do you have an official statement this is going to happen?

https://centos.org/distro-faq/#q6-will-there-be-separateparallelsimultaneous-streams-for-8-9-10-etc

**Comment by carlwgeorge** [comment]
> Ok, thanks. @opoplawski can you please change the name to `centos-stream-8-<arch>`?

Yes please, the distro is CentOS Stream, and the version is 8.  I know the distgit branches are currently named c8s, but that was some regex technical debt and shouldn't be used to draw any conclusions.  `centos-stream-8-<arch>` is the best name to use for mock build roots.

**Comment by opoplawski** [comment]
Renamed.

File: mock-core-configs/etc/mock/templates/centos-stream-8.tpl [hunk 1]
```diff
@@ -57,63 +57,63 @@ gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Officia
 
 [Stream-base-debuginfo]
 name=CentOS-Stream - Debuginfo
-baseurl=http://debuginfo.centos.org/8-stream/$basearch/
+baseurl=http://debuginfo.centos.org/stream-8/$basearch/
```
Comment (praiskup) [inline mock-core-configs/etc/mock/templates/centos-stream-8.tpl:60]: These repositories give me error 404.  Should they stay in the `8-stream` format?

Comment (carlwgeorge) [reply mock-core-configs/etc/mock/templates/centos-stream-8.tpl:60]: We haven't started publishing debuginfo packages for CS8 yet.  An 8-stream directory exists, but it's empty.  That will likely be the correct location once we do start publishing debuginfo packages.

File: mock-core-configs/etc/mock/templates/centos-stream-8.tpl [hunk 2]
```diff
@@ -4,7 +4,7 @@ config_opts['releasever'] = '8'
 config_opts['package_manager'] = 'dnf'
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
 config_opts['bootstrap_image'] = 'registry.centos.org/centos:8'
-config_opts['dnf_vars'] = { 'stream': '8-stream',
+config_opts['dnf_vars'] = { 'stream': 'stream-8',
```
Comment (carlwgeorge) [inline mock-core-configs/etc/mock/templates/centos-stream-8.tpl:7]: This variable should stay `8-stream`, as it's used by the mirrorlist service.

File: mock-core-configs/etc/mock/templates/centos-stream-8.tpl (hunk 1 above)
Comment (carlwgeorge) [reply mock-core-configs/etc/mock/templates/centos-stream-8.tpl:60]: In fact all the baseurls should stick with `8-stream` for now.

File: mock-core-configs/etc/mock/templates/centos-stream-8.tpl (hunk 2 above)
Comment (arrfab) [reply mock-core-configs/etc/mock/templates/centos-stream-8.tpl:7]: yes, that's what the Stream came with and as mirrorlist will pick path appearing on mirror network, let's stick with 8-stream (as that's also how it's done on Stream itself for mirrorlist .. let's not confuse more :)

File: mock-core-configs/etc/mock/templates/centos-stream-8.tpl (hunk 1 above)
Comment (praiskup) [reply mock-core-configs/etc/mock/templates/centos-stream-8.tpl:60]: Yep, that's what I meant.

**Comment by opoplawski** [comment]
Ah, well now we know where I got the idea to name things "8-stream" in the first place.  Hopefully things are correct now.

**Review by carlwgeorge — APPROVED** [review]

**Review by Conan-Kudo — APPROVED** [review]

**Comment by praiskup** [comment]
thank you

**Pull request merged**